supply-chain-security/compromises/2019/webmin-backdoor.md

```diff
@@ -0,0 +1,13 @@
+# Webmin backdoor
+
+At some time in April 2018, the Webmin development build server was exploited and a vulnerability added to one of the scripts. The vulnerable code was only present in Webmin packages offered for download via SourceForge, but not on GitHub. Because the timestamp of the modified script was set back, it did not show up in any Git diffs. The backdoor was first introduced in version 1.890 and was also included in 1.900 and 1.920.
+
+
+## Impact
+
+The attack surface have been enormous - without taking machines managed through Webmin into account. On its [GitHub page](https://github.com/webmin/webmin), the Webmin team claims their application has "over 1,000,000 installations worldwide." As of 19.08.2019 a [Shodan](https://www.shodan.io/) search query returns over 215,000 public Webmin instances, which can be attacked without needing to compromise internal networks or to bypass firewalls to reach a Webmin installation.
```

Comment:
I'm not sure what you mean by "without taking machines managed through Webmin into account"?
What's the source of the Shodan query? Probably worth citing.